test/common/testlib.py

```diff
@@ -1623,8 +1623,8 @@ def setUp(self, restrict: bool = True) -> None:
                 self.sshd_socket = 'sshd.socket'
         self.restart_sshd = f'systemctl try-restart {self.sshd_service}'
 
-        # only enabled by default on released OSes; see pkg/shell/manifest.json
-        self.multihost_enabled = image.startswith(("rhel-9", "centos-9")) or image in ["ubuntu-2204", "ubuntu-stable", "debian-stable", "fedora-39", "fedora-40", "fedora-coreos"]
+        # enabled by default except on RHEL 10 and friends.
+        self.multihost_enabled = image not in ["rhel-10", "centos-10"]
```

Comment:
Same thing -- this was quite deliberately a list of "old" OSes which shrinks in teh future instead of grows.

Reply:
I have restored the behavior of main.